stac_fastapi/api/stac_fastapi/api/__init__.py

```diff
@@ -1 +1,6 @@
 """api submodule."""
+from typing import Final
+
+from stac_fastapi.api.config import Settings
+
+settings: Final = Settings()
```

Comment:
why do we need this ?

Reply:
The goal is to have a single reference to `settings` that can be both accessed by the application and modified by tests for setup / teardown. That single reference could be implemented elsewhere, e.g. in config.py, but this approach is a personal preference for me as it makes clear that `settings`'s purview is the entire `stac_fastapi.api` package. Happy to change if desired.

Reply:
Looks like I have to move it anyway for `make docs-image` to succeed. It's failing because pydantic is not available when the build first tries to import the stac_fastapi.api package.